src/index.ts

```diff
@@ -50,28 +50,29 @@ export class ERC725 {
 
   options: {
     schema;
-    address;
-    providerType: ProviderType | null;
+    address?;
+    providerType?: ProviderType | null;
     provider?;
     ipfsGateway: string;
   };
 
-  constructor(schema: Erc725Schema[], address?: string, provider?: any) {
+  constructor(schema: Erc725Schema[], address?: string, provider?: any, ipfsGateway?: string) {
```

Comment:
the last object should be an options object.

Reply:
ipfs gateway is handled as a string currently... we could update this in the future(?)

Reply:
This is now in a separate issue.